src/commands/dev.ts

```diff
@@ -250,7 +263,21 @@ export async function command(commandOptions: CommandOptions) {
         }
         return `/web_modules/${spec}.js`;
       });
+
+      if (!fileLoc.includes('web_modules') && !fileLoc.includes('node_modules')) {
+        const imports = await scanCodeImportsExports(builtFileResult.result);
+        for (const imp of imports.reverse()) {
+          let spec = builtFileResult.result.substring(imp.s, imp.e);
+          if (imp.d > -1) {
+            const importSpecifierMatch = spec.match(/^\s*['"](.*)['"]\s*$/m);
+            spec = importSpecifierMatch![1];
+          }
+
+          addToDependents(spec, fileLoc);
```

Comment:
not handled: what happens when an import is removed? I think it would stay in the tree